components/setting_picture.test.tsx

```diff
@@ -27,9 +28,13 @@ describe('components/SettingItemMin', () => {
         onFileChange: () => {}, // eslint-disable-line no-empty-function
         updateSection: () => {}, // eslint-disable-line no-empty-function
```

Comment:
You can get rid of comment by using `jest.fn()`
```suggestion
        onFileChange: jest.fn(),
        updateSection: jest.fn(),
```